todo_app/data/trello_items.py

```diff
@@ -0,0 +1,86 @@
+from flask import session
+from ..classes.item_class import Item
```

Comment:
Very minor, but I'd generally advise using absolute imports over relative ones - there's a bit of justification in [the PEP8 style guide](https://peps.python.org/pep-0008/#imports) and lots of threads on Stack Overflow with various opinions if you happen to want to read more!
```suggestion
from todo_app.classes.item_class import Item
```